spot_driver/test/include/spot_driver/mock/mock_state_client.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2024 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#pragma once
+
+#include <gmock/gmock.h>
+
+#include <bosdyn/api/robot_state.pb.h>
+#include <spot_driver/api/state_client_interface.hpp>
+#include <string>
```

Comment:
```suggestion
```

Reply:
This is the one that needs to stay to satisfy IWYU

Reply:
sorry. i guess i missed that